README.md

```diff
@@ -20,51 +20,20 @@ For inspecting the running system inside of minikube with docker, run:
 eval $(minikube docker-env)
 ```
 
-
-## Building custom docker image
-
-The kubernetes sidecar injector plugin is not bundled in the Kong 1.0 docker image. Build and tag it with:
-
-```
-docker build -t kong .
-```
-
-You can then upload this to your docker repository with `docker push`, or if you can the above `minikube docker-env` command this will be loaded into your local kubernetes cluster.
-
-
 ## Setup Kong
 
 ```
-kubectl create namespace kong
-kubectl label namespace kong kong-sidecar-injection=disabled
-
-### Create a key+certificate for the control plane
-cat <<EOF | kubectl create -f -
-apiVersion: certificates.k8s.io/v1beta1
-kind: CertificateSigningRequest
-metadata:
-  name: kong-control-plane.kong.svc
-spec:
-  request: $(openssl req -new -nodes -batch -subj /CN=kong-control-plane.kong.svc | base64 | tr -d '\n')
-  usages:
-  - digital signature
-  - key encipherment
-  - server auth
-EOF
-kubectl certificate approve kong-control-plane.kong.svc
-kubectl -n kong create secret tls kong-control-plane.kong.svc --key=privkey.pem --cert=<(kubectl get csr kong-control-plane.kong.svc -o jsonpath='{.status.certificate}' | base64 --decode)
-kubectl delete csr kong-control-plane.kong.svc
-rm privkey.pem
-
-### Start database
-kubectl -n kong apply -f resource_definitions/postgres.yaml
-
-### Start control plane
-kubectl apply -f resource_definitions/kong-control-plane.yaml
+### Run Kong on k8s
+git clone https://github.com/Kong/kong-dist-kubernetes.git
+cd kong-dist-kubernetes
+git checkout feat/kong-1.1
```

Comment:
todo: remove this after https://github.com/Kong/kong-dist-kubernetes/pull/73 merges